### ISSUE TITLE
Add support for new Energenie Thermostat MIHO069

### ISSUE THREAD

**Issue by whaleygeek** [whaleygeek/pyenergenie#94]
https://energenie4u.co.uk/catalogue/product/MIHO069

From the product page:

The MiHome Thermostat is part of Energenie' s heating range of products and part of the MiHome home automation range. The heating range includes the radio controlled MiHome thermostat, and radio controlled Smart Radiator Valves, available separately. MiHome products link directly to the MiHome Gateway [available separately] which provides the communications between your MiHome device and the internet. The MiHome thermostat is suitable for switching any standard domestic boiler (see product specification below). The MiHome Thermostat connects to the boiler with a simple 2-wire connection and is battery powered. On-board controls to adjust temperature and turn the thermostat on/off are provided. The thermostat also senses occupancy and humidity which can be viewed in the App.

CODE: MIHO069

**Comment by Achronite** [comment]
Hi - I know you have not provided support for this yet, but do you know the internal productId of this?  It is missed off in devices.py and I would like to support it in my node-red nodes.

**Comment by whaleygeek** [comment]
Sorry, no idea. I am waiting for a loan device from Energenie so that I can power it up and see what comes back in the payload.

**Comment by whaleygeek** [comment]
I now have a loan device. I plan to investigate this soon...

![IMG_20190515_113800](https://user-images.githubusercontent.com/4086785/57771338-54f17780-770a-11e9-8c5c-77a7b0e37f2f.jpg)
![IMG_20190515_114011](https://user-images.githubusercontent.com/4086785/57771343-57ec6800-770a-11e9-8a71-b4a43f0ec903.jpg)
![IMG_20190515_114646](https://user-images.githubusercontent.com/4086785/57771347-5a4ec200-770a-11e9-9704-6d956e311b00.jpg)

**Comment by whaleygeek** [comment]
@jmigreen FYI

**Comment by Achronite** [comment]
bump

**Comment by whaleygeek** [comment]
Yeah, sorry, just been super busy. Next opportunity I get I will plug it in and sniff the device id and update the devices.py
